bentoml/deployment/utils.py

```diff
@@ -32,3 +34,23 @@ def generate_bentoml_deployment_snapshot_path(service_name, service_version, pla
         service_version,
         datetime.now().isoformat(),
     )
+
+
+def process_docker_api_line(payload):
+    """ Process the output from API stream, throw an Exception if there is an error """
+    # Sometimes Docker sends to "{}\n" blocks together...
+    for segment in payload.decode("utf-8").split("\n"):
+        line = segment.strip()
+        if line:
+            try:
+                line_payload = json.loads(line)
+            except ValueError as e:
+                print("Could not decipher payload from Docker API: " + e)
+            if line_payload:
+                if "errorDetail" in line_payload:
+                    error = line_payload["errorDetail"]
+                    sys.stderr.write(error["message"])
+                    raise RuntimeError("Error on build - code %s" % error["code"])
+                elif "stream" in line_payload:
+                    # TODO: move this to logger.info
```

Comment:
try just use logger.info here? and similarly for the logging.error above?

Reply:
done